src/pages/storage/forms/DuplicateVolumeForm.tsx

```diff
@@ -0,0 +1,270 @@
+          project:
+            values.project !== volume.project ? volume.project : undefined,
+          location: volume.location,
+        },
+      };
+
+      const existingVolumeLink = (
+        <Link
+          to={`/ui/project/${volume.project}/storage/pool/${volume.pool}/volumes/custom/${volume.name}`}
+          onClick={(e) => e.stopPropagation()}
+        >
+          <ItemName item={volume} />
+        </Link>
+      );
+
+      const targetProject =
+        values.project === formik.initialValues.project ? "" : values.project;
+
+      const targetMember =
+        values.member === formik.initialValues.member ? "" : values.member;
+
+      duplicateStorageVolume(payload, values.pool, targetProject, targetMember)
+        .then((operation) => {
+          toastNotify.info(`Duplication of volume ${volume.name} started.`);
+          eventQueue.set(
+            operation.metadata.id,
+            () => notifySuccess(values.name, values.project, values.pool),
+            (msg) =>
+              toastNotify.failure(
+                "Volume duplication failed.",
+                new Error(msg),
+                existingVolumeLink,
+              ),
+          );
+          close();
+        })
+        .catch((e) => {
+          toastNotify.failure(
+            "Volume duplication failed.",
+            e,
+            existingVolumeLink,
+          );
+        });
+    },
+  });
+
+  const disableClusterMemberSelector =
+    formik.values.project !== volume.project
+      ? "Can not migrate cluster member when changing project"
+      : undefined;
+
+  const disableProjectSelector =
+    isClustered && formik.values.member !== volume.location
+      ? "Can not change project when migrating cluster member"
+      : undefined;
+
+  return (
+    <Modal
+      close={close}
+      className="duplicate-instances-modal"
+      title="Duplicate volume"
+      buttonRow={
+        <>
+          <Button
+            appearance="base"
+            className="u-no-margin--bottom"
+            type="button"
+            onClick={close}
+          >
+            Cancel
+          </Button>
+          <ActionButton
+            appearance="positive"
+            className="u-no-margin--bottom"
+            loading={formik.isSubmitting}
+            disabled={!formik.isValid || projectsLoading || volumesLoading}
+            onClick={() => void formik.submitForm()}
+          >
+            Duplicate
+          </ActionButton>
+        </>
+      }
+    >
+      <Form onSubmit={formik.handleSubmit}>
+        <Input
+          {...formik.getFieldProps("name")}
+          type="text"
+          label="New volume name"
+          error={formik.touched.name ? formik.errors.name : null}
+        />
+        <ClusterMemberSelector
+          {...formik.getFieldProps("member")}
+          id="member"
+          label="Target cluster member"
+          disableReason={disableClusterMemberSelector}
+        />
```

Comment:
Depending on the storage pool driver the volume lives on, we should not show this selector. I.e. for a ceph backed storage, the cluster member has no effect.

I suspect the volume.location to be a good indicator -- if it is not an empty string, then we should show the selector.

Reply:
There is a related bug on the storage volume overview, if the volume.location is empty, we should render a "-" instead of nothing.

Reply:
This is a great observation! I think `volume.location` is likely a good indicator. Although, I'd like to do some additional testing by setting up a ceph pool, will report back.

Reply:
Yes it is. I encountered it on testing with a ceph pool on my local microcloud in VMs.

Reply:
Aaah okay I see, thanks for testing, then I think let's hide the selector if `volume.location` is not set for the existing volume, and the api query we send will only have the `project` param.

Reply:
@edlerd played around with this a little, I think we shouldn't hide the cluster member selector. In the case if we want to duplicate the volume to a non-remote storage pool, cluster member selection remains valid. Instead, I think we should disable the cluster member selector depending on the target pool selected. According to the [documentation](https://documentation.ubuntu.com/lxd/en/latest/explanation/storage/), we can determine if a pool is remote depending on the driver type i.e. all `ceph` types and `powerflex`. wdyt?

Reply:
Sounds good to me.

Reply:
great, that's already implemented :) let me know if you see anything wrong with it

Reply:
Should we then completely remove the cluster member selector from the duplicate form?

Reply:
yeah that's probably better, simpler :+1: . I don't think cluster location for a custom volume is as important as storage pool or project anyway :thinking: